### PR TITLE
:bug: jira-permission-validator: do not log sporadic 401 jira server errors

### PR DESCRIPTION
**Pull request by chassing**
Those 401 messages are just annoying in #sd-appsre-reconcile.

### CODE REVIEW

File: reconcile/jira_permissions_validator.py [hunk 1]
```diff
@@ -166,12 +166,16 @@ def board_is_valid(
                 )
                 error |= ValidationError.INVALID_PRIORITY
     except JIRAError as e:
-        if e.status_code != 403:
+        if e.status_code == 401:
+            # sporadic 401 errors, retrying
+            pass
```
Comment (rporres) [inline reconcile/jira_permissions_validator.py:171]: I'd add at least a debug message in case, at some point this stops being intermittent.

Comment (chassing) [reply reconcile/jira_permissions_validator.py]: Done

**Comment by rporres** [comment]
once fixed the failing test, this LGTM :)

**Comment by chassing** [comment]
> once fixed the failing test, this LGTM :)

those are shiny details ;) Fixed in the last commit

**Review by rporres — APPROVED** [review]

**Review by cubismod — APPROVED** [review]
LGTM

**Pull request merged**